foxtrot-core/src/test/java/com/flipkart/foxtrot/core/querystore/actions/FilterActionTest.java

```diff
@@ -779,7 +779,7 @@ public void testMissingIndicesQuery() throws FoxtrotException {
                 .indices()
                 .getIndex(new GetIndexRequest())
                 .actionGet();
-        assertEquals(3, Arrays.stream(response.getIndices())
+        assertEquals(4, Arrays.stream(response.getIndices())
```

Comment:
Why is this change needed now? The root cause needs to be checked. Test assertions should not be changed to make tests pass. The whole reason of assertions is to catch issues. If the assertion itself is changed, then there is no point of writing tests by the original authors.

Reply:
We have a new index for cardinality being created. So, number of default indices increased from 3 to 4